src/App.js

```diff
@@ -0,0 +1,71 @@
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { ToastContainer } from 'react-toastify';
+import LoginPage from './components/login/Login-page';
+import store from './app/store';
+import Motorcycle from './components/motorcycle/Motorcycle';
+import Detail from './components/motorcycle/Detail';
+import DeleteMotorcycle from './components/motorcycle/DeleteMotorcycle';
+import Layout from './layout/Layout';
+import AddMotorcycle from './components/motorcycle/AddMototorcycle';
+import AddReservation from './components/AddReservation/AddReservation';
+import Reservation from './components/Reservation/Reservation';
+// import Sidebar from './components/Sidebar/Sidebar';
+
+const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <LoginPage />,
+  },
+  {
+    path: '/motorcycles',
+    element: <Layout />,
+    children: [
+      {
+        path: '/motorcycles',
+        element: <Motorcycle />,
+      },
+      {
+        path: '/motorcycles/:id',
+        element: <Detail />,
+        loader: async ({ params }) => params,
+      },
+      {
+        path: '/motorcycles/new',
+        element: <AddMotorcycle />,
+      },
+      {
+        path: '/motorcycles/delete-motorcycles',
+        element: <DeleteMotorcycle />,
+        loader: async ({ params }) => params,
+      },
+    ],
+  },
+  {
+    path: '/reservations',
+    element: <Layout />,
+    children: [
+      {
+        path: '/reservations',
+        element: <Reservation />,
+      },
+      {
+        path: '/reservations/new',
+        element: <AddReservation />,
+      },
+    ],
+  },
+]);
+
+function App() {
```

Comment:
- [ ] When I go to the "Delete Motorcycle" page, the frontend enters an endless loop of requests to the server:

![image](https://github.com/TamaraDimov/vespa-rentals-frontend/assets/57421823/95d9b55b-6187-4d9a-af72-2f71c387e630)

In order for it to stop, I need to get out of the "Delete Motorcycle" section.

Most likely, this `useEffect` hook needs to be corrected:

![image](https://github.com/TamaraDimov/vespa-rentals-frontend/assets/57421823/eef3fedc-4cab-45ad-8df1-eb309c7559aa)


Kindly, make it so the motorcycles are fetched only once when a user visits that section of your project.

- [ ] Most likely due to the previous issue, the "Delete" button does not work, preventing me from deleting motorcycles.

Kindly fix this issue so your users can delete motorcycles.  👍